internal/handlers/tigris/tigrisdb/databases.go

```diff
@@ -74,7 +74,7 @@ func (tdb *TigrisDB) createDatabaseIfNotExists(ctx context.Context, db string) (
 					zap.String("db", db), zap.Error(err), zap.Duration("delay", delay),
 				)
 
-				ctxutil.Sleep(ctx, delay)
+				ctxutil.SleepWithJitter(ctx, delay, int64(i+1))
```

Comment:
The delay is also randomised value here, you would want to use `retryDelayMax` instead of `delay`.